plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/GitHook.kt

```diff
@@ -66,28 +79,29 @@ internal fun generateGitHook(
     gradleRootDirPrefix: String
 ) =
     """
-
+    
     CHANGED_FILES="${'$'}(${generateGitCommand(gradleRootDirPrefix)} | awk '$1 != "D" && $NF ~ /\.kts?$/ { print $NF }')"
-
+    
     if [ -z "${'$'}CHANGED_FILES" ]; then
         echo "No Kotlin staged files."
         exit 0
     fi;
-
+    
     echo "Running ktlint over these files:"
     echo "${'$'}CHANGED_FILES"
     
     git stash push --keep-index
     
     ${generateGradleCommand(taskName, gradleRootDirPrefix)}
-
+    
     echo "Completed ktlint run."
+    
     ${postCheck(shouldUpdateCommit)}
     
     git stash pop
     
     echo "Completed ktlint hook."
-
+    
```

Comment:
A bunch of indentation was added here. Can you trim this out?

Reply:
Some lines already have indentation, do you want me to remove it from all lines or leave as is?

Reply:
Can you trim the new indents?